### ISSUE TITLE
BREAKING CHANGE is too greedy

### ISSUE THREAD

**Issue by bcoe** [conventional-changelog/conventional-changelog#428]
the BREAKING CHANGE is greedy, in that it consumes text until the end of the commit message after the `:`; this perhaps makes sense for the body? but for the footer which can contain multiple lines of meta information, we should probably only consume tokens until the end of one line of the footer message if we see a BREAKING CHANGE.

**Comment by stevemao** [comment]
I agree since we want to put more meta data in footer, we should redefine how to parse it properly 👍

**Comment by jheaff1** [comment]
Any update on this? BREAKING CHANGE is still greedy 😁

**Comment by thislooksfun** [comment]
This is still a problem. I have a commit with a `BREAKING CHANGE` that I want to add a GitHub [`Co-Authored-By:`](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) trailer to. However, if I have the `Co-Authored-By` trailer *after* the `BREAKING CHANGE` it is included in the change logs, and if I have it *before* it only gets picked up if the `BREAKING CHANGE` is a single line (and uses the hyphenated version, `BREAKING-CHANGE`, instead). This means that I can't actually add the `Co-Authored-By` trailer to the same commit as the `BREAKING CHANGE`. Thankfully in this specific case I was able to break it into two commits, but this is still a use-case that should be supported.

**Comment by Starkie** [comment]
As thislooksfun mentioned, a workaround is to move the breaking changes section to the end of the commit message.
